main.go

```diff
@@ -32,10 +32,10 @@ func NewMigrationRunner(path string) *MigrationRunner {
 	return &MigrationRunner{path: path}
 }
 
-func (m *MigrationRunner) Run(ctx context.Context, conn *sql.Conn, lastRanId int) error {
+func (m *MigrationRunner) Run(ctx context.Context, conn *sql.Conn, lastRanId int) (error, int) {
```

Comment:
It's best-practice in go to always have `error` be the last return variable, so this would be: `(int, error)`.